cranelift/interpreter/src/interpreter.rs

```diff
@@ -119,13 +119,16 @@ impl Interpreter {
     /// Interpret a single [Block] in a [Function].
     fn block(&self, frame: &mut Frame, block: Block) -> Result<ControlFlow, Trap> {
         debug!("Block: {}", block);
-        for inst in frame.function.layout.block_insts(block) {
+        let layout = &frame.function.layout;
+        let mut maybe_inst = layout.first_inst(block);
+        while let Some(inst) = maybe_inst {
             match self.inst(frame, inst)? {
-                ControlFlow::Continue => continue,
+                ControlFlow::Continue => maybe_inst = layout.next_inst(inst),
                 ControlFlow::ContinueAt(block, old_names) => {
+                    debug!("Block: {}", block);
```

Comment:
Should we use a `trace!` here to be consistent with the other PR?

Reply:
Sure, added a commit for that.